### PR TITLE
Add skills to projects

### LINKED ISSUE #249: Add editable skills for projects

### PR DESCRIPTION
**Pull request by wdoug**
This PR closes #249 

#### What does this PR do?

Adds editable skills to projects

### CODE REVIEW

File: src/sections/ProjectSection.js [hunk 1]
```diff
@@ -34,4 +45,19 @@ const ProjectSection = ({ project, editing, onFormDataChange }) => {
   );
 };
 
+ProjectSection.fragments = {
```
Comment (dchao19) [inline src/sections/ProjectSection.js:48]: These are really nice - maybe at some point we should consider moving these into their own query file as the data is used throughout the application.

File: src/sections/ProjectSection.js [hunk 2]
```diff
@@ -1,10 +1,12 @@
 import React from 'react';
+import gql from 'graphql-tag';
 import EditableText from '../forms/EditableText';
 import EditableMarkdown from '../forms/EditableMarkdown';
+import EditableSkills from '../forms/EditableSkills';
 import ProjectIssues from './ProjectIssues';
 
 const ProjectSection = ({ project, editing, onFormDataChange }) => {
```
Comment (dchao19) [inline src/sections/ProjectSection.js:8]: This whole edit project section needs some styling, but the functionality is still there and it doesn't undermine this PR I don't think

**Review by dchao19 — APPROVED** [review]
Everything looks good! Definitely some things to tidy up in the future though, but its nice to finally start somewhere. Thanks!

File: src/sections/ProjectSection.js (hunk 1 above)
Comment (wdoug) [reply src/sections/ProjectSection.js:48]: I definitely think we can use fragments a lot better. I'm not sure how I feel about putting them in their own query files though as that seems to have less of the original benefit of graphql of co-locating queries next to components.

**Pull request merged**